src/views/create/index.tsx

```diff
@@ -112,7 +113,7 @@ const Create = (_: RouteComponentProps) => {
                                     setStep(step - 1);
                                 }}
                                 onNext={() => {
-                                    if (!userData) {
+                                    if (!isSignedIn) {
```

Comment:
Once we nave the userData again, we can use it here as it was.

Reply:
I'd suggest we keep it as `isSignedIn`, this is functionally the same, [see here](https://github.com/fungible-systems/micro-stacks/blob/main/packages/react/src/hooks/use-auth.ts#L36)